src/main/java/mekanism/common/tile/TileEntityElectrolyticSeparator.java

```diff
@@ -646,4 +646,21 @@ public boolean hasCapability(Capability<?> capability, EnumFacing side)
 	{
 		return new Object[] {fluidTank, leftTank, rightTank};
 	}
+
+	@Override
+	public void recalculateUpgradables(Upgrade upgrade)
+	{
+		super.recalculateUpgradables(upgrade);
```

Comment:
Wouldnt it be better to have this in the `default` block, to save doing any useless calcs of energy use?